edalize/xcelium.py

```diff
@@ -76,55 +71,55 @@ def get_doc(cls, api_ver):
         if api_ver == 0:
             return {'description' : "Xcelium simulator from Cadence Design Systems",
                     'lists' : [
-                        {'name' : 'xmvhdl_options',
+                        {'name' : 'elab_options',
                          'type' : 'String',
-                         'desc' : 'Additional options for compilation with xmvhdl'},
-                        {'name' : 'xmvlog_options',
+                         'desc' : 'Additional run options for xrun -elaboration'},
+                        {'name' : 'xrun_prefix',
                          'type' : 'String',
-                         'desc' : 'Additional options for compilation with xmvlog'},
-                        {'name' : 'xmsim_options',
+                         'desc' : 'prefix for using submission servers like bjob or nc'},
+                        {'name' : 'run_options',
                          'type' : 'String',
-                         'desc' : 'Additional run options for xmsim'},
-                        {'name' : 'xrun_options',
-                         'type' : 'String',
-                         'desc' : 'Additional run options for xrun'},
+                         'desc' : 'Additional run options for xrun -R'},
                         ]}
```

Comment:
The new options look cleaner, but I'm worried about current users of the xcelium backend because removing the old options will break things for them. Or do you mean that the xcelium backend didn't work at all before? In that case I guess there is less risk anyone will complain

Reply:
unless someone would use the generated make as standalone without the generated file list - the previous backend did not work.
so I think it is safe to assume that no one used it.